packit_service/worker/jobs.py

```diff
@@ -233,6 +237,65 @@ def service_config(self):
             self._service_config = ServiceConfig.get_service_config()
         return self._service_config
 
+    def report_task_accepted(
```

Comment:
glad you refactored the code into a new method

long term we should start breaking down jobs.py even further and come up with a clearer structure